mu-plugins/_loader.php

```diff
@@ -14,4 +14,5 @@
 }
 unset( $live_domain, $subdomain, $domain_specific_file, $type_specific_file );
 
+require_once(WPMU_PLUGIN_DIR . '/relevanssi/relevanssi.php');
```

Comment:
Avoid using `require_once` like a function with parenthesis. Instead, use it as a statement like the  line below.